.gitignore

```diff
@@ -31,3 +34,4 @@ common.pyc
 /gh-pages/_site
 /*compiler*.jar
 /local_blockly_compressed_vertical.js
+
```

Comment:
Just wondering why there's a random extra newline here.

Reply:
Please remove the stray newline here.